### ISSUE TITLE
GUI Limitations Frame Control

### ISSUE THREAD

**Issue by dmrvb** [WaynePhillipsEA/twinbasic#797]
**Describe the bug**
1 Frame control is non functional, it does not contain other controls, in particular it does not "group" option buttons.
(I know this is also reported on VBforums)

2 On a hi resolution monitor the outline box is very thin (one pixel?) and so quite difficult to see.

The longstanding VB6 bug with controls in frames not displaying correctly has returned with a vengence in Win11.
I assume the tB frame will work correctly (please) ?

**Comment by WaynePhillipsEA** [comment]
Yes, (1) is not implemented yet.   We can keep this issue open to track that.

For (2) we would probably have to custom draw the frame in order to change that.  Perhaps @Kr00l can advise if that's right.

**Comment by Kr00l** [comment]
The VB6 bug for themed Frame controls (originally just happening in Win XP) can be fixed by redirecting WM_PRINTCLIENT, WM_MOUSELEAVE to DefWindowProc.

About the thin borders. In my VBCCR I do actually ownerdraw the Frame control. But of course the border is the same as in the real Frame control. (using uxtheme API)
I just ownerdraw it in VBCCR to make it more vb-ish. Thus able to colorize the text and allow to make text alignment. Also it allows to position and align a picture.
Example: (center text alignment, right picture alignment and red color text)
![image](https://user-images.githubusercontent.com/33756666/163134297-9b7212b6-d4f8-49bc-9952-4e22fbe6c1af.png)

**Comment by dmrvb** [comment]
> The VB6 bug for themed Frame controls (originally just happening in Win XP) can be fixed by redirecting WM_PRINTCLIENT, WM_MOUSELEAVE to DefWindowProc.
> 
> About the thin borders. In my VBCCR I do actually ownerdraw the Frame control. But of course the border is the same as in the real Frame control. (using uxtheme API) I just ownerdraw it in VBCCR to make it more vb-ish. Thus able to colorize the text and allow to make text alignment. Also it allows to position and align a picture. Example: (center text alignment, right picture alignment and red color text) ![image](https://user-images.githubusercontent.com/33756666/163134297-9b7212b6-d4f8-49bc-9952-4e22fbe6c1af.png)

Yes, i've just looked and the frame outline is also very thin in VB6. 
Might it make sense to build your controls into tB? I appreciate that its a lot of complicated code and might need some looking after. Your fix is interesting (I don't fully understand it) but that will require me to subclass lots of my forms??. I will probably just change all of the frames to pictureboxes. It would be great if tB was ready in time but I need to fix things in the next few weeks.
